examples/fluids/src/setupstg_shur14.c

```diff
@@ -0,0 +1,331 @@
+ * as the only two entries, separated by a single space
+ *
+ * @param[in] comm MPI_Comm for the program
+ * @param[in] path Path to the file
+ * @param[out] nrows Number of rows
+ */
+static PetscErrorCode GetNRows(const MPI_Comm comm,
+                               const char path[PETSC_MAX_PATH_LEN], int *nrows) {
+  PetscErrorCode ierr;
+  const int char_array_len = 512;
+  int dims[2];
+  FILE *fp;
+
+  PetscFunctionBeginUser;
+  ierr = OpenPHASTADatFile(comm, path, char_array_len, dims, &fp); CHKERRQ(ierr);
+  *nrows = dims[0];
+  ierr = PetscFClose(comm, fp); CHKERRQ(ierr);
+  PetscFunctionReturn(0);
+}
+
+/*
+ * @brief Read the STGInflow file and load the contents into stg_ctx
+ *
+ * Assumes that the first line of the file has the number of rows and columns
+ * as the only two entries, separated by a single space.
+ * Assumes there are 14 columns in the file
+ *
+ * Function calculates the Cholesky decomposition from the Reynolds stress
+ * profile found in the file
+ *
+ * @param[in] comm MPI_Comm for the program
+ * @param[in] path Path to the STGInflow.dat file
+ * @param[inout] stg_ctx STGShur14Context where the data will be loaded into
+ */
+static PetscErrorCode ReadSTGInflow(const MPI_Comm comm,
+                                    const char path[PETSC_MAX_PATH_LEN], STGShur14Context stg_ctx) {
+  PetscErrorCode ierr;
+  int ndims, dims[2];
+  FILE *fp;
+  const int char_array_len=512;
+  char line[char_array_len];
+  char **array;
+
+  PetscFunctionBeginUser;
+
+  ierr = OpenPHASTADatFile(comm, path, char_array_len, dims, &fp); CHKERRQ(ierr);
+
+  CeedScalar rij[6][stg_ctx->nprofs];
+  CeedScalar *prof_dw = &stg_ctx->data[stg_ctx->offsets.prof_dw];
+  CeedScalar *eps = &stg_ctx->data[stg_ctx->offsets.eps];
+  CeedScalar *lt = &stg_ctx->data[stg_ctx->offsets.lt];
+  CeedScalar (*ubar)[stg_ctx->nprofs] = (CeedScalar (*)[stg_ctx->nprofs])
+                                        &stg_ctx->data[stg_ctx->offsets.ubar];
+
+  for (int i=0; i<stg_ctx->nprofs; i++) {
+    ierr = PetscSynchronizedFGets(comm, fp, char_array_len, line); CHKERRQ(ierr);
+    ierr = PetscStrToArray(line, ' ', &ndims, &array); CHKERRQ(ierr);
+    if(ndims < dims[1]) SETERRQ(comm, -1,
+                                  "Line %d of %s does not contain enough columns (%d instead of %d)", i,
+                                  path, ndims, dims[1]);
+
+    prof_dw[i] = (CeedScalar) atof(array[0]);
+    ubar[0][i] = (CeedScalar) atof(array[1]);
+    ubar[1][i] = (CeedScalar) atof(array[2]);
+    ubar[2][i] = (CeedScalar) atof(array[3]);
+    rij[0][i]  = (CeedScalar) atof(array[4]);
+    rij[1][i]  = (CeedScalar) atof(array[5]);
+    rij[2][i]  = (CeedScalar) atof(array[6]);
+    rij[3][i]  = (CeedScalar) atof(array[7]);
+    rij[4][i]  = (CeedScalar) atof(array[8]);
+    rij[5][i]  = (CeedScalar) atof(array[9]);
+    lt[i]      = (CeedScalar) atof(array[12]);
+    eps[i]     = (CeedScalar) atof(array[13]);
+
+    if(prof_dw[i] < 0) SETERRQ(comm, -1,
```

Comment:
Same thing  here with funky formatting - If you haven't run `make style`, I'd do so. If you have and these files weren't fixed, then I can investigate.

Reply:
Yeah, these files aren't being hit by `make style` for some reason. Running `astyle --options=.astylerc` on them explicitly results in changes, but `make style` doesn't.

Reply:
Only thing I see different about these files is the underscore in the filename? The glob in the `Makefile` looks like it should handle that fine, but I'm not familiar with what special rules `make` may have for globbing.

Reply:
The recursing deep enough into folders has been flaky, so I suspect that's the culprit

Reply:
Should we consider merging in-progress stuff and switching to clang-format here, as we did in Ratel? (Should check in with other libceed stakeholders before taking that step.)

Reply:
Personally, I like the output better and find clang-format easier to use/configure so :+1: from me

Reply:
Should I manually run with astyle real quick, or leave it as is and wait for clang-format to come in? Granted, if the culprit is make globbing, I don't know if switching to clang-format necessarily fixes it.

Reply:
Swapping to clang-format will take time. I'd go ahead and format this.

Recursing into folders doesn't seem work right with astyle, but it seemed to work fine with clang-format. Not really a make issue.

Reply:
Upon further investigation, make is correctly passing the file to astyle (verified via `make style-c -n`), _and_ is successfully running on the file. Astyle just 
A) doesn't care about the missing space after the `if` and
B) enforces shifting the `SETERRQ` arguments out of alignment (ie. making them aligned manually is undone by astyle).

The former can be enforced with `--pad-header`. Not sure about the latter though.